lib/cylc/cylc_subproc.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" A wrapper function to aggregate these calls in one file, with logging for
+    what is calling it and the commands given
+    Bandit B602: subprocess_popen_with_shell_equals_true
```

Comment:
This comment is inaccurate - no logging being done here.

Reply:
ah yes it requires matts changes on master. I'll remove this